core/src/main/java/com/backblaze/b2/json/B2Json.java

```diff
@@ -139,23 +140,35 @@ public static B2Json get() {
         }
     }
 
+    public void toJson(Object obj, OutputStream out) throws IOException, B2JsonException {
+        toJson(obj, B2JsonOptions.DEFAULT, out);
+    }
+
+    public void toJson(Object obj, B2JsonOptions options, OutputStream out) throws IOException, B2JsonException {
+        toJson(obj, options, out, null);
+    }
```

Comment:
How about javadocs for these two methods?  They are public methods that users of the library will be calling.

Reply:
Done